lib/src/core/verifier/sample/policy.rs

```diff
@@ -0,0 +1,49 @@
+use super::*;
+use anyhow::Result;
+use std::path::PathBuf;
+use tokio::fs;
+
+impl Sample {
+    pub fn opa_path(&self) -> Result<PathBuf> {
```

Comment:
Do you really need this method? Seems like you could just calculate the path directly below.